recipe/meta.yaml

```diff
@@ -20,6 +20,8 @@ outputs:
         strong:
           - cudatoolkit {{ cuda_compiler_version }}|{{ cuda_compiler_version }}.*
```

Comment:
```suggestion
          - cudatoolkit {{ cuda_compiler_version }}|{{ cuda_compiler_version }}.*
          - __glibc >=2.17   # [linux]
```

Reply:
Well we only need this if CUDA 11.0 is being used. Should that be in the selector as well then?

Reply:
yes

Reply:
Ok this should be fixed now. Please take another look.